uaclient/tests/test_cli_config.py

```diff
@@ -0,0 +1,57 @@
+import mock
+import pytest
+
+from uaclient.cli import main
+
+M_PATH = "uaclient.cli."
+
+HELP_OUTPUT = """\
+usage: pro config <command> [flags]
+
+Manage Ubuntu Pro configuration
+
+Flags:
+  -h, --help  show this help message and exit
+
+Available Commands:
+  
+    show      show all Ubuntu Pro configuration setting(s)
+    set       set Ubuntu Pro configuration setting
+    unset     unset Ubuntu Pro configuration setting
+"""  # noqa
+
+
+@mock.patch("uaclient.cli.logging.error")
+@mock.patch("uaclient.cli.setup_logging")
+@mock.patch(M_PATH + "contract.get_available_resources")
+class TestMainConfig:
```

Comment:
This test was moved from `test_cli_config_show` as it is not specific to `show`.